bower.json

```diff
@@ -49,6 +49,6 @@
     "rx-angular": "rx.angular#^1.1.3"
   },
   "resolutions": {
-    "patternfly-sass": "~3.4.0"
+    "angular": "1.5.5"
```

Comment:
why is this change needed? isn't angular already part of miq? and why pf is removed?